.ci.yaml

```diff
@@ -1791,8 +1791,6 @@ targets:
       tags: >
         ["devicelab", "android", "linux"]
       task_name: hot_mode_dev_cycle_linux__benchmark
-    runIf:
```

Comment:
I'm not going to check these .ci.yaml changes in, I just want to see if the desktop tests pass in presubmit.